pkg/login/kubeConfig.go

```diff
@@ -37,21 +37,18 @@ func CreateClusterKubeConfig(clusterID string, kubeConfig api.Config) (string, e
 
 	// Write kube config if file not exist
```

Comment:
A small change! Please change this comment as this PR is creating a new kubeconfig file even if it exits

Reply:
Oups, indeed ;)